src/components/TodoInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { completed, title } = todo;
+
+  return (
+    <>
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+        />
+      </label>
+
+      <span className="todo__title">
+        {title}
+      </span>
+      <button type="button" className="todo__remove">×</button>
+
+      <div className="modal overlay">
```

Comment:
Consider removing or adjusting the modal overlay and loader, as they may not be necessary for this component.